src/app/item-popup/DesktopItemActions.tsx

```diff
@@ -339,6 +348,55 @@ function MoveLocations({
   );
 }
 
+function PullButtons({
+  item,
+  itemOwner,
+  submitMoveTo,
+  vault,
+}: {
+  item: DimItem;
+  itemOwner: DimStore<DimItem>;
+  submitMoveTo: MoveSubmit;
+  vault?: DimStore<DimItem>;
+}) {
+  const showAmounts = item.maxStackSize > 1 || item.bucket.hash === BucketHashes.Consumables;
+  const moveAllLabel = showAmounts ? `All` : undefined;
```

Comment:
i18n?

Reply:
committed but missed my final push. should be good now